tests/torch/sparsity/movement/test_structured_mask.py

```diff
@@ -438,36 +418,5 @@ def test_detect_supported_model_family(self, run_recipe: BaseMockRunRecipe):
         _, compressed_model = create_compressed_model(run_recipe.model(),
                                                       empty_nncf_config,
                                                       dump_graphs=False)
-        retval = detect_supported_model_family(compressed_model)
-        if run_recipe.supports_structured_masking:
-            assert retval == run_recipe.model_family
-            assert retval in STRUCTURED_MASK_STRATEGY.registry_dict
-        else:
-            assert retval is None
-
-    @pytest.mark.parametrize('run_recipe', STRUCTURED_MASK_SUPPORTED_RECIPES)
-    def test_create_strategy(self, run_recipe: BaseMockRunRecipe):
```

Comment:
Some tests are not needed after deletion structured mask strategy @vuiseng9 @yujiepan-work